### PR TITLE
chore: Remove endpoints dependency on meteor/check

### PR DESCRIPTION
**Pull request by matheusbsilva137**
## Further comments

TC-1090

### CODE REVIEW

File: apps/meteor/app/api/server/v1/permissions.ts [hunk 1]
```diff
@@ -40,13 +39,9 @@ API.v1.addRoute(
 
 API.v1.addRoute(
 	'permissions.update',
-	{ authRequired: true },
+	{ authRequired: true, permissionsRequired: ['access-permissions'] },
```
Comment (ggazzo) [inline apps/meteor/app/api/server/v1/permissions.ts:42]: not related please move to a different pr

File: apps/meteor/app/api/server/v1/channels.ts [hunk 2]
```diff
@@ -947,7 +944,12 @@ API.v1.addRoute(
 
 API.v1.addRoute(
 	'channels.list',
-	{ authRequired: true },
+	{
+		authRequired: true,
+		permissionsRequired: {
+			GET: { permissions: ['view-c-room', 'view-joined-room'], operation: 'hasAny' },
```
Comment (ggazzo) [inline apps/meteor/app/api/server/v1/channels.ts:950]: same

**Review by ggazzo — CHANGES_REQUESTED** [review]
since this is supposed to be huge because there is no other way to go, avoid different changes, stick with meteor/check removal, if you can split into commits/pr each endpoint would be great also

**Comment by matheusbsilva137** [comment]
Closed in favor of #31579 and #31576
Thanks for the reviews, @ggazzo ! Please check the new PRs later

**Pull request closed without merging**